python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml

```diff
@@ -1425,6 +1429,175 @@ keys:
+                groups:
+                  type: list
+                  items:
+                    type: str
+                    description: |-
+                      Trunk group name.
+                      Warning: This should not be combined with `ethernet_ineterfaces[].trunk_groups`.
+            dot1q:
+              type: dict
+              keys:
+                ethertype:
+                  type: int
+                  convert_types:
+                    - str
+                  min: 1536
+                  max: 65535
+                  description: Ethertype/TPID (Tag Protocol IDentifier) for VLAN tagged frames.
+                vlan_tag:
+                  type: str
+                  description: Allow/disallow VLAN tagged Frames.
+                  valid_values:
+                    - disallowed
+                    - required
+            source_interface:
+              type: str
+              description: |-
+                tx: Allow bridged packets to go out of the source interface.
+                tx multicast: Allow multicast packets only to go out of the source interface.
+              valid_values:
+                - tx
+                - tx multicast
+            access_vlan:
+              type: int
+              convert_types:
+                - "str"
+              min: 1
+              max: 4094
+              description: |-
+                Set VLAN when interface is in access mode.
+                Warning: This should not be combined with `ethernet_interfaces[].mode = access/dot1q-tunnel`.
+            phone:
+              type: dict
+              keys:
+                vlan:
+                  type: int
+                  min: 1
+                  max: 4094
+                  convert_types:
+                    - str
+                  description: |-
+                    Warning: This should not be combined with `ethernet_interfaces[].phone.vlan`.
+                trunk:
+                  type: str
+                  description: |-
+                    Warning: This should not be combined with `ethernet_interfaces[].phone.trunk`
+                  valid_values: ["tagged", "tagged phone", "untagged", "untagged phone"]
+            mode:
+              type: str
+              valid_values: ["access", "dot1q-tunnel", "trunk", "trunk phone"]
+              description: |-
+                Warning: This should not be combined with `ethernet_interfaces[].mode`
+            pvlan_mapping:
+              type: str
+              convert_types:
+                - int
+              description: |-
+                Secondary VLAN IDs of the private VLAN mapping.
+                Warning: This should not be combined with `ethernet_interfaces[].pvlan_mapping`.
+            vlan_translations:
+              description: |-
+                Warning: This should not be combined with `ethernet_interfaces[].vlan_translations`.
+              type: list
+              primary_key: direction
```

Comment:
May be we need to allow duplicate primary keys.

Reply:
s1-leaf1(config-if-Et2)#sh active 
interface Ethernet2
   description P2P_LINK_TO_S1-SPINE1_Ethernet2
   mtu 1500
   switchport mode trunk
   switchport vlan translation 100 102
   switchport vlan translation in 102 103
   switchport vlan translation in 104 105
   
   EOS support multiple commands for a direction.

Reply:
Got to know few more updates on vlan translations, changing the schema for the same.

Reply:
pushed the updated schema